src/js/controller/model.js

```diff
@@ -260,7 +260,12 @@ define([
             return this[attr];
         },
         'set' : function(attr, val) {
+            if (this[attr] === val) {
+                return;
+            }
             this[attr] = val;
+            this.trigger('change:'+attr, this, val);
+            this.trigger('change', this);
```

Comment:
It would be nice if the model only triggered attribute changes. We could then use 'all' for all changes.